state-chain/runtime/src/chainflip/bitcoin_block_processor.rs

```diff
@@ -191,12 +195,9 @@ impl Hook<HookTypeFor<TypesVaultDepositWitnessing, RulesHook>> for TypesVaultDep
 impl Hook<HookTypeFor<TypesEgressWitnessing, RulesHook>> for TypesEgressWitnessing {
 	fn run(
 		&mut self,
-		(block, age, block_data): (BlockNumber, Range<u32>, EgressBlockData),
+		(block, age, block_data, safety_margin): (BlockNumber, Range<u32>, EgressBlockData, u32),
 	) -> Vec<(BlockNumber, BtcEvent<TransactionConfirmation<Runtime, BitcoinInstance>>)> {
-		if age.contains(
-			&(u64::steps_between(&0, &BitcoinIngressEgress::witness_safety_margin().unwrap_or(0)).0
```

Comment:
👍  - worth adding a comment on the ingress-egress pallet storage item here, to explain that it won't be used for any chains using the elections witnessing (just so anyone debugging doesn't get confused)